fastlane_bot/data/multichain_addresses.csv

```diff
@@ -104,12 +104,13 @@ carbon_v1,linea,carbon_v1,0x0000000000000000000000000000000000000000,0x000000000
 agni_v3,mantle,uniswap_v3,0x25780dc8Fc3cfBD75F33bFDAB65e969b603b2035,0x319B69888b0d11cEC22caA5034e25FfFBDc88421,,110692,
 butter_v3,mantle,uniswap_v3,0xEECa0a86431A7B42ca2Ee5F479832c3D4a4c2644,0xAe5c0a73bAE513fe5842DbcB8E32fc7E3e30DA57,,22966090,
 carbon_v1,mantle,carbon_v1,0x7900f766F06e361FDDB4FdeBac5b138c4EEd8d4A,0x7900f766F06e361FDDB4FdeBac5b138c4EEd8d4A,,18438182,
-carbon_v1,mantle,carbon_v1,0x3749f6Ef09B0f74CD5324126b02d867A03DEE844,0x3749f6Ef09B0f74CD5324126b02d867A03DEE844,,55241318,
 cleopatra_v2,mantle,solidly_v2,0xAAA16c016BF556fcD620328f0759252E29b1AB57,0xAAA45c8F5ef92a000a121d102F4e89278a711Faa,,34705175,
 cleopatra_v3,mantle,uniswap_v3,0xAAA32926fcE6bE95ea2c51cB4Fcb60836D320C42,0xAAAE99091Fbb28D400029052821653C1C752483B,,34705175,
+fusionx_supernova,mantle,carbon_v1,0x04FBC7f949326fFf7Fe4D6aE96BAfa3D8e8A8c0a,0x04FBC7f949326fFf7Fe4D6aE96BAfa3D8e8A8c0a,,61955475,
```

Comment:
`fusionx_supernova` doesn't appear anywhere in the code other than this specific line (in contrast with, for example, `puffs_penthouse_v3`).

Is that expected, or is there possibly some missing infrastructure required for the bot to operate over `fusionx_supernova`?

Reply:
That is expected. As a Carbon deployment, it does not include any static data or require any deployment-specific handling.